### PR TITLE
feat(mode): card component integration

### PR DESCRIPTION
**Pull request by theJohnnyMe**
1. integrating card component

### CODE REVIEW

File: react/src/App.tsx [hunk 1]
```diff
@@ -78,6 +78,22 @@ function App() {
...
+              body-img="https://scdn.reflex.prod.aws.scania.com/thumbnails/2.6a4c8c88169b49c419d1a8/18.12bc286418679e166a96d74.webp?Expires=1677920403&Key-Pair-Id=K2MBMFK028C481&Signature=IqfnGEZSkb7OlgIsYOrN7nhHK5D-yqRyE~Q4McUtv8th76lygq9dZ-bMQMRx8G4fO39m7oS761xLXx-bAkqkVxbG1s1WF5iB3yqr81h-ayBy1gQfyZWM0QX-de151xjT-WEIpw6g8DcAJ5vzz4h~VfeYjbOzd~E8WSJstSYzsxMJXL16J0lXpKpz4T~0dIkA11nXai47offlgut1kvxY1Rd0lsr~cT2tNf7lSlCECXo9OfOeLwfYk7l3Q1AybdcHhm9iQQjam0uujI242k7XPbQ2XDSvK7mEGF86XXQwU9VaLRB7KqKsb3UJPB9ugLXy1l4wyQ05GMW8ysRR9mznfA__"
+              clickable
+          >
+            <div slot="card-body">Here is a short description in body text</div>
+            <div slot="card-bottom">
```
Comment (nathalielindqvist) [inline react/src/App.tsx:92]: The body image is not visible for me, does that matter?
image

File: react/src/App.tsx [hunk 2]
```diff
@@ -78,6 +78,22 @@ function App() {
           <sdds-breadcrumb-item current={true}>Current page</sdds-breadcrumb-item>
         </sdds-breadcrumb>
       </div>
+        <div>
+          <div className="sdds-headline-02 sdds-u-pb1">Card</div>
+          <sdds-card
```
Comment (nathalielindqvist) [inline react/src/App.tsx:83]: Should we have the native version in this example as well? :)

File: react/src/App.tsx (hunk 1 above)
Comment (theJohnnyMe) [reply react/src/App.tsx:147]: This should be fixed right now, probably someone removed the image from Reflex.

File: react/src/App.tsx (hunk 2 above)
Comment (theJohnnyMe) [reply react/src/App.tsx:138]: I put it as a topic for today's dev check-in, I wonder the same.
If so, makes sense to have a tab on the demo page where just HTML&CSS components are.

**Review by mJarsater — APPROVED** [review]
LGTM! Could add the native one also if you want to. both otherwise, merge it!

**Comment by theJohnnyMe** [comment]
> LGTM! Could add the native one also if you want to. both otherwise, merge it!

I would like to explore the idea of another page in this APP maybe we will have only native ones. I put that as a topic for dev check in.

**Pull request merged**